### PR TITLE
added name to font-hack

### PR DESCRIPTION
**Pull request by caarlos0**
refs #437

### CODE REVIEW

File: Casks/font-hack.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 cask 'font-hack' do
   version '2.018'
   sha256 'f4a610ae049073845736cbee2c2cd1e87982a2af582839398caab8edd11e4fa0'
+  name 'Hack'
```
Comment (vitorgalvao) [inline Casks/font-hack.rb]: Needs to be under `url`.

**Comment by caarlos0** [comment]
@vitorgalvao done

**Comment by vitorgalvao** [comment]
Thank you. Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by caarlos0** [comment]
@vitorgalvao also done

**Pull request merged**